dash/dash-renderer/src/observers/executedCallbacks.ts

```diff
@@ -50,21 +53,24 @@ const observer: IStoreObserverDefinition<IStoreState> = {
             if (!itempath) {
                 return false;
             }
-
             // This is a callback-generated update.
-            // Check if this invalidates existing persisted prop values,
-            // or if persistence changed, whether this updates other props.
-            updatedProps = prunePersistence(
-                path(itempath, layout),
-                updatedProps,
-                dispatch
-            );
+            let props = updatedProps;
+
+            if (id === '_pages_content') {
```

Comment:
Shouldn't this be the opposite `id !== '_pages_content'` for it to record apply the persistence?
In any case the comments here are now confusing, might be worth rewriting them with the actual logic of what it does here and how it get there.